programmers/creat-strange-characters.js

```diff
@@ -0,0 +1,25 @@
+function solution(s) {
+    let altrnChar; 
+    let altrnCollection;
+    s = s.split(" ");
+    for (let i = 0; i < s.length; i++) {
+        altrnCollection = [];
+        for (let j = 0; j < s[i].length; j++) {
+            if (j % 2 === 0) {
+                altrnChar = s[i][j].toUpperCase();
+            } else {
+                altrnChar = s[i][j].toLowerCase();
+            }
+        altrnCollection.push(altrnChar);
+        }
+        altrnCollection = altrnCollection.join("");
```

Comment:
배열로 push하고 join하고해서 문자열로 다시 만드시는데 처음부터 문자열을 이용하셔서 푸시는게 효율이 좋습니다.
```suggestion
    for (let words of s.split(" ")) {
        for (let i = 0; i < words.length; i++) {
            result += i % 2 ? words[i].toLowerCase() : words[i].toUpperCase();
        }
        result += " "
    }
```